nvdaHelper/ISimpleDOM_sconscript

```diff
@@ -24,5 +24,10 @@ tlbFile,headerFile,iidSourceFile,proxySourceFile,dlldataSourceFile=env.TypeLibra
 	source=idlFile,
 	MIDLFLAGS=['/c_ext','/I',Dir('.')],
 )
+# hack: Ignore midl.exe when deciding to rebuild, as its position in the dependencies

```

Comment:
Do we want to open an issue somewhere to keep track of this? It might be helpful for someone who wants to investigate, and see what has been tried. Having the issue number in the comments really helps people track down where the information is.